packages/flutter/lib/src/widgets/form.dart

```diff
@@ -355,15 +321,6 @@ class FormField<T> extends StatefulWidget {
   /// {@endtemplate}
   final AutovalidateMode autovalidateMode;
```

Comment:
Maybe we need to remove references to the old API in the comments as well

Reply:
Thank you!